include/Graph/Graph.hpp

```diff
@@ -0,0 +1,123 @@
+#pragma once
+#include <iostream>
+#include <queue>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+class Layer {
+ private:
+  int id;
+  std::string name;
+  int type;
+  std::string version;
+  int numInputs;
+  int numNeurons;
+  std::vector<int> primer;
+
+ public:
+  Layer() : id(0), type(1) {}
+  Layer(int id1, int type1) : id(id1), type(type1) {}
+  int checkID() { return id; }
+  void In(std::vector<int> a) { primer = a; }
+  void Work() {
+    switch (type) {
+      case 1:
+        for (size_t i = 0; i < primer.size(); ++i) {
+          primer[i] += 1;
+        }
+        break;
+      case 2:
+        for (size_t i = 0; i < primer.size(); ++i) {
+          primer[i] *= 2;
+        }
+        break;
+    }
+  }
```

Comment:
This is also part of the testing, it should not be inside the class

Reply:
I deleted it